includes/wikis.php

```diff
@@ -125,13 +125,13 @@
 		'api' => $baseurl . '/brawlstars/api.php',
 	],
 	'starcraft' => [
-		'name' => 'Brood War',
+		'name' => 'StarCraft: Brood War',
 		'theme-light' => '#3a5ba9',
 		'theme-dark' => '#b0c5ff',
 		'api' => $baseurl . '/starcraft/api.php',
 	],
 	'wildrift' => [
-		'name' => 'Wild Rift',
+		'name' => 'League of Legends: Wild Rift',
```

Comment:
Keeping this one as just Wild Rift is probably better for SEO purposes, I don't think anyone actually ever calls it by its full name either.

Reply:
I am no expert in SEO, but I'd disagree on this one. This is really used in every official communication (e.g. https://www.youtube.com/watch?v=v9S8DHRy6AM). It also shows, that Wild Rift is really a LoL Mobile. But again, I cannot estimate the impact on SEO.

Reply:
Our expert suggests leaving this one as Wild Rift